ethereum/erc20/view_token_balances_latest.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE VIEW erc20.view_token_balances_latest AS
+SELECT distinct on (wallet_address, token_address)
+wallet_address,
+token_address,
+t.symbol as token_symbol,
+amount_raw,
+amount_raw / 10^coalesce(t.decimals, 0) amount_formatted,
+amount_raw / 10^coalesce(t.decimals, 0) * p.price amount_usd,
+timestamp
+FROM erc20.token_balances
+left join erc20.tokens t on t.contract_address = token_address
+left join prices.usd p on p.contract_address = token_address and p.minute = date_trunc('hour', NOW())
```

Comment:
Note: here the price for `NOW()` is fetched for the current hour, so it becomes for example `2021-06-24 15:00:00`

Reply:
The timestamp is for the last transfer while the price is current? That seems a little weird to me. Might be better to remove timestamp or have it as the same as the price?

https://duneanalytics.com/queries/68312

Reply:
Probably good idea to just remove it. Was thinking about naming it something else but I think since price is there it will be confusing, can be added later if needed.

Reply:
hmm I think it could be useful. Could maybe name it `last_transfer_timestamp` and then `current_usd_value`. Maybe @0xBoxer has any thoughts?

Reply:
`last_transfer_timestamp` seems like something people can make good use of.
IF we can agree on that name I don't think there is a need to rename the `amount_usd` column.

Reply:
I'd name it usd#1